### ISSUE TITLE
Bookmarks

### ISSUE THREAD

**Issue by Angular-Angel** [Agora-Project/agora-meteor#6]
Add a feature for bookmarking posts. Perhaps a list on the left, that can be minimized, and that posts can be dragged into? Or just an option on the posts under more to bookmark.

**Comment by saulrh** [comment]
There's a user story for an "automatically advancing" bookmark that makes it easier to follow a discussion. Say you're having a discussion; you make a bookmark to keep track of it. But every time someone posts you need to manually add it to the bookmark so you're not continually opening up chunks of discussion manually when you go back to it. There should be a reasonable way to automate this process so the bookmark automatically advances with the conversation.

**Comment by saulrh** [comment]
UI Suggestion: instead of drag-drop, maybe colors and a lasso tool? Click "Edit bookmark", posts are colored blue or left uncolored depending on inclusion in bookmark, click to toggle, drag a loop around, when it closes, everything inside it is added/removed from the bookmark.

**Comment by Angular-Angel** [comment]
Once SAA has been implemented, I'm going to look into using it to create a method of tracking the currently active "heads" of a conversation, so people can see where the discussion is at.

**Comment by saulrh** [comment]
Right, that's what was bugging me. This is something like git, we'll want something akin to branches/tags to take care of things like new chapters of a fanfiction being posted.

**Comment by Angular-Angel** [comment]
Mmm, exactly.
